### DISCUSSION TITLE
为什么要选择github，而不是Facebook这种更加成熟更加专业的社交平台？

### DISCUSSION THREAD

**Discussion by HKLHaoBin** [share121/inter-knot#2678] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/27fb9141bd6d9a72a5ebff8b6ae10a00.png" alt="黑塔 你说咩啊" width="205" >

其实我有个疑问，为什么要选择？github的社区聊天功能，实现绳网的论坛，而不是选择Facebook这种更加成熟更加专业的。
如果说选择github 是为了他的高门槛的话。好像Facebook也是要那么高的门槛才能进入。

而且如果不考虑做扩展开发的话，那Facebook的话它能提供更成熟的算法，能按时间排序，也能按热度排序了，你们觉得呢？

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="105" >

**Comment by homteam** [comment]
很有实施的可能，这个委托，我不会

**Comment by IKUN05648** [comment]
Facebook我记得要外网或VPN的，而github有时候不要

**Reply by HKLHaoBin** [reply]
我尝试过不用梯子去发评论，好像是发不出来的

**Reply by jsishsjsj** [reply]
> 我尝试过不用梯子去发评论，好像是发不出来的

嘿嘿嘿～发不发得出来是要看脸的，像我这次就是脸白发出来了。

**Reply by dnmdngcsb** [reply]
facebook是被墙了，github是DNS污染，不一样的形式，所以github可以用加速器，或者干脆直接发就行，但是facebook和SNS这种就要VPN

**Comment by huangminzhe** [comment]
脸书要外网，GitHub其实不开加速器也很慢

**Comment by NekoWise** [comment]
最主要的原因是，Facebook僅支援他們的界面，并不支援像繩網一樣的第三方界面

**Reply by HKLHaoBin** [reply]
是的

**Comment by KongBai000-0** [comment]
脸书更偏向大众一些，git更偏程序交流（偏向不同）

**Comment by ysd1911** [comment]
如果用了facebook的话，更麻烦吧

**Comment by Maple013** [comment]
github毕竟算是正经且知名的程序网站，想进来的话只需要WT这样的加速器，不算麻烦；但如果用Facebook，要翻墙，那规模一大就有可能大完特完了。

**Reply by miaochu1** [reply]
facebook要翻墙，来着的基本都是国内，大规模翻墙聚集感觉容易出事

**Comment by Xmug** [comment]
因为作者是为了绳网这个界面而包的饺子

**Comment by AnonCN** [comment]
使用github好像是因为可以直接将代码放入github中托管运行？
github.io

**Reply by HKLHaoBin** [reply]
主要是没有API

**Comment by KongBai000-0** [comment]
完了，新的问题，我用qq邮箱注册git的为什么你们写的回复会跑到我qq邮箱啊？？？

**Reply by wxgl** [reply]
github发送新消息通知很正常啊
